src/main/java/gov/cabinetofice/gapuserservice/web/LoginControllerV2.java

```diff
@@ -115,17 +114,25 @@ public RedirectView redirectAfterLogin(
         final String idToken = tokenResponse.getString("id_token");
         final String authToken = tokenResponse.getString("access_token");
 
-        oneLoginService.validateAuthTokenSignatureAndAlgorithm(authToken);
         IdTokenDto decodedIdToken = oneLoginService.decodeTokenId(idToken);
-        oneLoginService.validateIdToken(decodedIdToken);
+
+        if (!Objects.equals(this.configProperties.getProfile(), "LOCAL")) {
+            // FIXME fix mocks so we don't have to ignore validation locally
+            oneLoginService.validateIdToken(decodedIdToken);
+            oneLoginService.validateAuthTokenSignatureAndAlgorithm(authToken);
+        }
 
         final StateCookieDto stateCookieDto = oneLoginService.decodeStateCookie(stateCookie);
         final String redirectUrl = stateCookieDto.getRedirectUrl();
 
         verifyStateAndNonce(decodedIdToken.getNonce(), stateCookieDto, state);
 
         final OneLoginUserInfoDto userInfo = oneLoginService.getOneLoginUserInfoDto(authToken);
-        oneLoginService.validateUserSub(decodedIdToken.getSub(), userInfo.getSub());
+
+        if (!Objects.equals(this.configProperties.getProfile(), "LOCAL")) {
+            // FIXME fix mocks so we don't have to ignore validation locally
```

Comment:
Should remove the comment as it's an open source repo, could create a ticket for it instead?